cmd/cephcsi.go

```diff
@@ -68,6 +68,11 @@ func init() {
 	flag.StringVar(&conf.NodeID, "nodeid", "", "node id")
 	flag.StringVar(&conf.PluginPath, "pluginpath", defaultPluginPath, "plugin path")
 	flag.StringVar(&conf.StagingPath, "stagingpath", defaultStagingPath, "staging path")
+	flag.BoolVar(
+		&conf.SetMetadata,
+		"setmetadata",
+		false,
+		"set metadata on the volume")
```

Comment:
I think this won't exceed the line length limit. it can be a single statement like others,